test/e2e/network/dual_stack.go

```diff
@@ -64,7 +64,9 @@ var _ = common.SIGDescribe("[Feature:IPv6DualStack]", func() {
 
 			framework.ExpectEqual(len(internalIPs), 2)
 			// assert 2 ips belong to different families
-			framework.ExpectEqual(netutils.IsIPv4String(internalIPs[0]) != netutils.IsIPv4String(internalIPs[1]), true)
+			if netutils.IsIPv4String(internalIPs[0]) == netutils.IsIPv4String(internalIPs[1]) {
+				framework.Failf("both internalIPs %s and %s are IPv4", internalIPs[0], internalIPs[1])
+			}
```

Comment:
the asser checks that both IPs has different ip family, if fails if both are IPv4 or both are IPv6

Reply:
I had updated the output accordingly.